README.md

```diff
@@ -70,33 +70,43 @@ The React components are built using the [TypeScript](https://www.typescriptlang
 
 ## 💻 Getting started
 
-As a prerequisite to run this app you will need to install [Node.js](https://nodejs.org/en/download)(version 16.20.2 is currently used in local development and in the pipeline) and [npm](https://docs.npmjs.com/cli/v7/commands/npm-install):
+As a prerequisite to run this app you will need to install [Node.js](https://nodejs.org/en/download)(version 20.17.0 is currently used in local development and in the pipeline) and [npm](https://docs.npmjs.com/cli/v10/commands/npm-install):
 
-### Running locally
+If you have Node Version manager you can set node to the correct version using the nvm command.
 
 ```
-npm install
-npm run dev
+nvm use
 ```
 
-The app will attempt to start on port `3000` and should be available on localhost: [http://localhost:3000](http://localhost:3000)
-
-### Logging in
-
-The staff dashboard is using [LBH Google Auth](https://github.com/LBHackney-IT/LBH-Google-auth) for authentication. Permissions to what a user is authorised to do, is managed by mapping Google groups.
+### Running locally
 
-You need a **@hackney.gov.uk** Google account to sign in. Speak to Hackney IT if you don't have this.
+Copy the .env sample to the root of your application and get the variables from AWS params store or the password manager.
 
 Next, you need to tell your computer to run the app from a hackney.gov.uk domain. Add this line to your hosts file (Windows: `C:\Windows\System32\drivers\etc\hosts`, Mac: `/etc/hosts`):
 
 ```
 127.0.0.1   localdev.hackney.gov.uk
 ```
 
-Update the `APP_URL` variable in the `.env` file to match. When you next launch the app, it should be on `http://localdev.hackney.gov.uk:3000`.
+If necassary update the `APP_URL` variable in the `.env` file to match. When you next launch the app, it should be on `http://localdev.hackney.gov.uk:3000`.
 
 If you have the right configuration setup within the `.env` file, you should be able to access the staff dashboard.
 
+```
+npm install
+npm run dev
+```
+
+The app will attempt to start on port `3000` and should be available on localhost: [http://localhost:3000](http://localhost:3000)
```

Comment:
This line can probably be removed since the instructions above show how to run it correctly on `http://localdev.hackney.gov.uk:3000` in dev mode.